Sources/KeychainSwift.swift

```diff
@@ -125,12 +142,12 @@ open class KeychainSwift {
   */
   @discardableResult
   open func set(_ value: Bool, forKey key: String,
-    withAccess access: KeychainSwiftAccessOptions? = nil) -> Bool {
+    withAccess access: KeychainSwiftAccessOptions? = nil, withControlFlag controlFlag: KeychainSwiftAccessControlFlag? = nil) -> Bool {
   
     let bytes: [UInt8] = value ? [1] : [0]
     let data = Data(bytes)
 
-    return set(data, forKey: key, withAccess: access)
+    return set(data, forKey: key, withAccess: access,withControlFlag: controlFlag)
```

Comment:
```suggestion
    return set(data, forKey: key, withAccess: access, withControlFlag: controlFlag)
```

Reply:
Fixed in: 067bb33c34c39595b6b117e014665a5c661bfca4